<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Слот</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_slotName"/>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_slotType">
           <item>
            <property name="text">
             <string>INT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FLOAT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BOOL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>STRING</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_slotValue"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_saveSlot">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Поиск по значению слота</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Имя слота</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="action_addFrame"/>
    <addaction name="action_addSlot"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_addFrame">
   <property name="text">
    <string>Фрейм</string>
   </property>
  </action>
  <action name="action_addSlot">
   <property name="text">
    <string>Слот</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
